### DISCUSSION TITLE
Simultaneous working with one file

### DISCUSSION THREAD

**Discussion by SuperBoss9** [vrtmrz/obsidian-livesync#349] [General]
Hi All!

I'm looking for a solution that will enable my team to work with Obsidian more efficiently. So the core question here - is it possible to work (edit) the same file (just a note or a canvas) with at least two users at the same time? Will it work with LiveSync plugin?

One more question - is it possible to use IPv6 address for pointing to the database?

**Comment by vrtmrz** [comment]
Thank you for making the topic!
> is it possible to work (edit) the same file (just a note or a canvas) with at least two users at the same time? Will it work with LiveSync plugin?

Yes, Not perfect behaviour, but works! With a custom replicator [LiveSync-Bridge](https://github.com/vrtmrz/livesync-bridge), we can share some notes with some people.
If each modification is independent and could be merged, it would be merged automatically. (This is a feature of LiveSync).

> IPv6 address

I have not checked it yet, however, if we want to use LiveSync on mobile, we need a valid domain. Otherwise, we might use any address. Did you have some troubles?

**Reply by SuperBoss9** [reply]
>Yes, Not perfect behaviour, but works! With a custom replicator [LiveSync-Bridge](https://github.com/vrtmrz/livesync-bridge), we can share some notes with some people.
If each modification is independent and could be merged, it would be merged automatically. (This is a feature of LiveSync).

The idea is to share a vault in Obsidian with some sensitive data between a distributed team. Currently the data is shared via Syncthing. Everything works except ability to work in the vault all together especially under one document.

Additional questions:
a. As far as I understand LiveSync will not do automerge but LiveSync-Bridge will do. But I need to install Bridge at every computer and set it up before usage. Is it correct?
b. What Live-Sync/Bridge does if a non text file in the vault is changed? For example, users can update Excel file that is in the vault. Will it be corrupted?
c. Is work with Obsidian Canvas supported in terms of automerge or it is considered as a binary file?

>I have not checked it yet, however, if we want to use LiveSync on mobile, we need a valid domain. Otherwise, we might use any address. Did you have some troubles?

Haven't tested it yet. But IPv6 looks a healthy alternative to other different ways of connection like reverse proxy etc. Not for mobile but for desktops. If I point to an IPv6 address then I can connect to the server from everywhere. If it is inside my network then I will connect to it directly. If I'm outside of my network then I can use ordinary IPv6 connection to the server.
